traits/adaptation/adaptation_manager.py

```diff
@@ -319,7 +321,7 @@ def _get_applicable_offers(self, current_protocol, path):
 
         edges = []
 
-        for from_protocol_name, offers in self._adaptation_offers.items():
+        for from_protocol_name, offers in six.iteritems(self._adaptation_offers):
```

Comment:
I'd suggest just leaving this one as it is. I think it should be fine on both Python 2 and Python 3. (We end up computing an actual list in Python 2, and just iterating over a view in Python 3, but I don't think that matters.)

Reply:
I'll merge as is; we can fix later.